src/main/resources/application.yml

```diff
@@ -21,6 +21,11 @@ spring:
             client-secret: ${system.client.secret:clientsecret}
             authorization-grant-type: client_credentials
             scope: read
+          community-api:
+            provider: hmpps-auth
+            client-id: ${oauth.client.id}
```

Comment:
The creds here will need to be the same as the ones above `${system.client.id:make-recall-decision-api}` etc as far as I know